azure-pipelines.yml

```diff
@@ -5,15 +5,72 @@
 
 trigger:
 - master
+- releases/*
+
+variables:
+  PYTHON: "Conda" # use Julia's packaged Conda build for installing packages
+  CMDSTAN_VERSION: "2.21.0"
```

Comment:
2.22.0 is the latest version.

CmdStanPy has the following script to install the latest version

https://github.com/stan-dev/cmdstanpy/blob/master/cmdstanpy/install_cmdstan.py

Reply:
At the moment, CmdStan.jl still tests 2.21.0 with CI. Once it begins testing against 2.22.0, I'll happily update.